src/99-appendices/09-entities.md

```diff
@@ -133,13 +133,26 @@ Full name: Echo
 
 Format: `echo-<index>`
 
-Definition: Multi-echo data MUST be split into one file per echo.
-Each file shares the same name with the exception of the `_echo-<index>`
-key/value.
+Definition: If constituents of an entity-linked file collection differ as a
+function of `EchoTime` acquisition parameter, the `_echo-<index>` key/value
+pair MUST be used to distinguish individual files.
 Please note that the `<index>` denotes the number/index (in the form of a
 nonnegative integer) of the echo not the echo time value which needs to be
 stored in the field `EchoTime` of the separate JSON file.
 
+## flip 
```

Comment:
This file is generated from a script. You need to add `flip` to the schema and use that to generate this file instead. [This](https://github.com/bids-standard/bids-specification/blob/580183fe1af767b4ea7ee3b0e9459713c0c6b658/src/schema/entities.yaml) is the file that needs to be added, and [this](https://github.com/bids-standard/bids-specification/blob/580183fe1af767b4ea7ee3b0e9459713c0c6b658/tools/bids_schema.py#L456-L457) is the script you need to run to regenerate the `09-entities.md` file.

Reply:
@tsalo so sorry that I missed that giant header comment. Fixing right away.